app/javascript/autocomplete_toggle.js

```diff
@@ -0,0 +1,22 @@
+export default function initializeAutocompleteToggle() {
+  const searchField = document.querySelector('#search_field');
```

Comment:
Is there a reason you aren't using Stimulus for this?

Reply:
By making that choice, you are binding to selectors that are not guaranteed to be stable between versions of blacklight/arclight.  This seems more dangerous to me, even though it might be less code.

Reply:
This makes sense, thanks for the feedback. The team decided put this to draft for now as we learn more about what the stakeholders actually want from this feature.